meson.build

```diff
@@ -63,6 +63,11 @@ else
 endif
 
 subdir('data')
-subdir('plugins/gnome-builder')
 subdir('src')
-subdir('test')
+if get_option('plugins')
+  subdir('plugins/gnome-builder')
+endif
+
+if get_option('tests')
```

Comment:
There shouldn't be a need to override this, right? We don't install the tests anyway.

Reply:
Sure it is.

Reply:
What happens if you don't include this change? You can still build the package just fine, right?

Reply:
In fact, don't you only need the change to `plugins/gnome-builder/meson.build` to get this working?